fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -305,7 +307,28 @@ public void setLastRefreshTime(long lastRefreshTime) {
     // Maintenance plan for this MV
     private transient ExecPlan maintenancePlan;
 
-    public static class MvRewriteContext {
+    // The maximum delay duration is used to determine which partitions to refresh.
+    // During this duration, the current result of 'needToRefreshPartitionNamesCache' is used
+    // as the basis for 'needToRefreshPartitionNames'.
+    // Please note that the unit of measurement is in seconds.
+    @SerializedName(value = "maxMVRewriteStaleness")
+    private int maxMVRewriteStaleness = 0;
+
+    // The next timestamp, measured in seconds, determines when to check for
+    // partitions that need to be refreshed(unit: ms).
```

Comment:
unit is ms or second?
the comment is ambiguous

Reply:
fix it.